docs/src/modeler/time_series.md

```diff
@@ -0,0 +1,332 @@
+# Managing Time Series Data
+
+**Note**: This section assumes the reader is familiar with the definitions in the
+quick start guide
```

Comment:
If the definitions remain in the quick start guide then this should be defined as a link (relative paths work) so that the user can click on it.

Reply:
Yes, I will add those to the right section when I get the docs PR. I have it noted